assets/js/theme/global/modal.js

```diff
@@ -296,8 +296,29 @@ export function alertModal() {
 /*
  * Display the given message in the default alert modal
  */
-export function showAlertModal(message) {
+export function showAlertModal(message, options = {}) {
     const modal = alertModal();
+    const cancelBtn = modal.$modal.find('.cancel');
+    const { icon, showCancelButton, onConfirm } = options;
+
     modal.open();
+    modal.$modal.find('.alert-icon').hide();
+
+    if (icon === 'warning') {
+        modal.$modal.find('.warning-icon').show();
+    } else {
+        modal.$modal.find('.error-icon').show();
+    }
```

Comment:
Small proposal
We can make it more declarative using `else if` instead of `else`. It will make more understandable which `states` of icons (and modal overall) we have
```
    if (icon === 'warning') {
        modal.$modal.find('.warning-icon').show();
    } else if (icon === 'error') {
        modal.$modal.find('.error-icon').show();
    }
```

Reply:
@yurytut1993 done!